c2corg_ui/templates/i18n.html

```diff
@@ -376,5 +376,8 @@
 <span translate>has updated the article</span>
 <span translate>has updated the route</span>
 <span translate>has updated the image</span>
+<span translate>has added photos to the route</span>
+<span translate>has added photos to the outing</span>
+<span translate>has added photos to the waypoint</span>
```

Comment:
Ah yes, I remember. But did you try this? https://angular-gettext.rocketeer.be/dev-guide/annotate/#interpolation